app/middleware/setLocals.js

```diff
@@ -8,7 +9,11 @@ function fromRequest(req, res, next) {
     latitude: req.query.latitude,
     longitude: req.query.longitude,
   };
+  const fullUrl = fullUrlUtils(req);
   const backLink = backLinkUtils(req);
+  res.locals.fullUrl = {
```

Comment:
See comments about simplifying the returned value. If the suggestion is implemented this can be simplified to just `res.locals.completeOriginalUrl = completeOriginalUrl(req)`;

Reply:
Done